nci-crdc.datacommons.io/manifests/fence/fence-config-public.yaml

```diff
@@ -1,137 +1,14 @@
 S3_BUCKETS:
-  tcga-protected-dcf-databucket-gen3:
```

Comment:
I'm not sure why this has been deleted?

Reply:
This PR was created by a new Gen3-QA autoamtion (from our `gen3release-sdk`) so it blindly copies everything from preprod to prod.
That seems to be one exception I will need to accommodate in that PR-creation logic.

Thanks for checking!